src/gpu/intel/ocl/tile_ops.h

```diff
@@ -55,12 +55,16 @@ __attribute__((overloadable)) int local_atomic_max(local int *p, int v) {
             global type *p, type##n v) { \
         intel_sub_group_block_write##suffix##n( \
                 (global itype *)p, as_##itype##n(v)); \
+    } \
+    __attribute__((overloadable)) void block_store(local type *p, type##n v) { \
+        intel_sub_group_block_write##suffix##n( \
+                (local itype *)p, as_##itype##n(v)); \
     }
 
 #define DEF_BLOCK_LOAD_STORE1(type, itype, suffix) \
-    __attribute__((overloadable)) \
-            type##1 block_load(const global type *p, int vlen) __attribute__( \
-                    (enable_if(vlen == 1, "wrong vector length"))) { \
+    __attribute__((overloadable)) type##1 block_load( \
+            const global type *p, int vlen) \
+            __attribute__((enable_if(vlen == 1, "wrong vector length"))) { \
```

Comment:
Is there some clang-format version difference going on here? (seems like the code itself isn't changed)

Reply:
Looks like it. The version is CI is pinned to `11`.